user/mprocs/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "mprocs"
```

Comment:
this looks like it was added by mistake into the same commit

Reply:
oh ye